### ISSUE TITLE
client.servers.get_all() returns one created date for all servers

### ISSUE THREAD

**Issue by mambas** [hetznercloud/hcloud-python#66]
When I create new server via hcloud API and then get all servers information - 'created' date is set one for all. When I then get one server information - date is different
Python 3.6.8 (default, Oct  7 2019, 12:59:55)
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
#> from hcloud import Client
#> client = Client(token="sLgM*******N2wMh")
#> s = client.servers.get_all()
#> s[0].created
**datetime.datetime(2019, 10, 11, 10, 36, 49, tzinfo=tzutc())**
#> s[1].created
**datetime.datetime(2019, 10, 11, 10, 36, 49, tzinfo=tzutc())**
#> s1 = client.servers.get_by_id(3434180)
#> s1.created
datetime.datetime(2019, 10, 11, 10, 36, 49, tzinfo=tzutc())
#> s2 = client.servers.get_by_id(3434122)
#> s2.created
datetime.datetime(2019, 10, 11, 10, 27, 43, tzinfo=tzutc())

**Comment by LKaemmerling** [comment]
I can not reproduce this. I have tested it with the following snippet:
```
from hcloud import Client

client = Client(token="YOUR_API_TOKEN")

servers = client.servers.get_all()

for server in servers:
    onlyServer = client.servers.get_by_id(server.id)
    print("Server Name:" + server.name)
    if server.created == onlyServer.created:
        print("OK")
    else:
        print("get_all:" + server.created.strftime("%m/%d/%Y, %H:%M:%S"))
        print("get_by_id:" + onlyServer.created.strftime("%m/%d/%Y, %H:%M:%S"))
```

**Comment by muraty** [comment]
The same issue here. @LKaemmerling you can reproduce it with:
```
from hcloud import Client

client = Client(token='YOUR_API_TOKEN')

servers = client.servers.get_all()

print('## get_all')
for server in servers:
    print(server.id, '---->', server.created)

print('## get_by_id')
for server in servers:
    s = client.servers.get_by_id(server.id)
    print(s.id, '---->', s.created)
```

**Comment by muraty** [comment]
Additional information: I use the version ```hcloud==1.6.2```

**Comment by LKaemmerling** [comment]
@muraty  @mambas  a fix for this was released yesterday as `hcloud==1.6.3`

**Comment by muraty** [comment]
@LKaemmerling Just tested and it worked. Thanks!
